pkgs/tools/text/u2ps/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, ghostscript_headless
+}:
+
+stdenv.mkDerivation rec {
```

Let's use `finalAttrs`:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```